src/CFamily.UnitTests/SystemAbstractions/ProcessWrapperTests.cs

```diff
@@ -125,20 +126,22 @@ public void Execute_ProcessTimesOut_CanBeKilled()
             var exeName = WriteBatchFileForTest(TestContext,
 @"@echo Waiting for keyboard input which will not arrive...
 set /p arg=
+exit -999
 ");
 
             using var processScope = StartProcess(exeName);
             var testSubject = new ProcessWrapper(processScope.Process);
 
             // Should timeout be because the batch file is waiting for input...
-            testSubject.WaitForExit(2000);
+            testSubject.WaitForExit(1000);
             testSubject.HasExited.Should().Be(false);
             CheckProcessIsRunning(testSubject.Id);
 
             // Now kill the process
             testSubject.Kill();
             testSubject.HasExited.Should().Be(true);
             testSubject.ExitCode.Should().Be(-1);
+            System.Threading.Thread.Sleep(500);
```

Comment:
Added a short delay before checking the process has terminated (in case the problem was that the managed framework _Process_ class has finished it's work, but the OS hasn't finished clearing up the underlying native process).

If these tests continue to be flaky I'll [Ignore] them and create a ticket to investigate them later.